12-buster-slim/Dockerfile

```diff
@@ -26,7 +26,10 @@ RUN apt-get update -qq \
     && mkdir -p $NODE_MODULES_PATH && chown $SERVICE_USER:$SERVICE_USER $NODE_MODULES_PATH
 
 # Add a strict security policy for Imagemagick (note: Imagemagick 7 is currently not available in Debian)
-COPY imagemagick-policy.xml /etc/ImageMagick-6/policy.xml
+COPY policies/imagemagick-policy-6.xml /etc/ImageMagick-6/policy.xml
```

Comment:
Unsure about this change. It is nice to not have to copy this file when we create a new image, but if we need to make any changes, we'd need to create a new file. This also requires us to run our context at the root, which could effect Docker cache. Looking at some other official Docker images, the pattern seems to be keeping files with their respective Dockerfile.

Reply:
I can't currently speak to whether or not the root context affects the cache, so that's an unknown I'll have to concede to.